WalletWasabi.Gui/Global.cs

```diff
@@ -48,7 +49,17 @@ public class Global
 		public string DataDir { get; }
 		public TorSettings TorSettings { get; }
 		public BitcoinStore BitcoinStore { get; }
-		public HttpClientFactory HttpClientFactory { get; }
+
+		/// <summary>
+		/// HTTP client factory for communicating with the Wasabi backend.
+		/// </summary>
+		public HttpClientFactory BackendHttpClientFactory { get; }
+
+		/// <summary>
+		/// HTTP client factory for communicating with external third parties.
+		/// </summary>
+		public HttpClientFactory ExternalHttpClientFactory { get; }
```

Comment:
@kiminuo based on examining your BlockstreamInfo tests we separated `HttpClientFactory` to `BackendHttpClientFactory` (renamed) and `ExternalHttpClientFactory` (added) because things didn't work on regtest. If you have a more elegant concept in mind, then feel free to create another PR.